rudder-web/src/main/webapp/secure/administration/apiManagement.html

```diff
@@ -118,7 +118,15 @@ <h1 id="title" style="color: #000; wrap-word:break-word;">{{formTitle(myNewAccou
                       <div class="simplemodal-bottom">
                         <div class="popupButton" style="width: 98%;border-top: 1px solid #9A9A9A;margin-top: 10px;">
                           <span>
-                            <input type="submit" class="btn-primary" style="float:right;" value="Save" ng-disabled="newAccount.$pristine || newAccount.$invalid" ng-click="checkAndSaveAccount(myNewAccount,myNewAccount.index,newAccount);newAccount.$setPristine();" />
+                            <button
```

Comment:
Why changing from input to button fix it ? 

We should put the check in the ng-disable i think ... but htat for another time :)